canvases/dark-packages/main.dark

```diff
@@ -122,4 +171,33 @@ let _handler _req =
     |> unwrap
     |> String.toBytes
 
-  PACKAGE.Darklang.Stdlib.Http.response respBody 200
\ No newline at end of file
+  PACKAGE.Darklang.Stdlib.Http.response respBody 200
+
+
+[<HttpHandler("GET", "/category/:name")>]
+let _handler _req =
+  let fullName = name |> String.split "."
+  let owner = fullName |> List.head |> unwrap
+  let name = fullName |> List.last |> unwrap
+
+  let modules = fullName |> (List.drop 1) |> PACKAGE.Darklang.Stdlib.List.dropLast
+  let modulesWithoutItem = fullName |> (List.drop 1)
+
+  let getall = DB.getAll NamesDB
+
+  let res =
+    getall
```

Comment:
Could we please rename this to `allItems` or in-line this (and wrap in parens ()) ?